#!/usr/bin/env python
PACKAGE = "tub_feasibility_check"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

ee_tf_enum = gen.enum([gen.const("Palm", int_t, 0, "Sets the ee-tf to the palm"),
                       gen.const("Fingertips", int_t, 1, "Sets the ee-tf to the fingertips")],
                      "An enum to choose location of end effector transform")

gen.add("ee_tf", int_t, 0, "A size parameter which is edited via an enum", 0, 0, 1, edit_method=ee_tf_enum)

exit(gen.generate(PACKAGE, "tub_feasibility_check_dynamic_reconfigure", "KinematicsCheck"))
